src/app/home.js

```diff
@@ -46,7 +46,14 @@ const MenuItem = ({item}) => r(
     item.submenu && [SubMenu, {items: item.submenu}],
   ])
 
-export default (menu) => r(
+const Text = ({textWithLink}) => r(
+    ['p', {className: 'text__wrapper'},
+      textWithLink.descriptionBefore,
+      ['a', {href:textWithLink.link ?? '', className:'link__text'}, textWithLink.linkText],
+      textWithLink.descriptionAfter,
+    ])
+
+export default (menu, textWithLink) => r(
```

Comment:
I guess here, we don't need to be generic. Name the `textWithLink` semantically. For example `newHandbook` / `newHandbookProxy`